Netflix Clone/Models/CreateSessionResponse.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  CreateSessionResponse.swift
+//  Netflix Clone
+//
+//  Created by Polina Poznyak on 6.07.23.
+//
+
+import Foundation
+
+struct CreateSessionResponse: Codable {
+    let success: Bool
+    let session_id: String
```

Comment:
Usually In swift we use camelCase, while JSON uses snakeCase.
It's fine for the practice, but keep that in mind ⚠️